src/host_context.rs

```diff
@@ -368,10 +368,10 @@ mod test {
     }
 
     #[test]
-    fn vec_host_objs() {
+    fn vec_host_objs() -> Result<(), ()> {
         let mut host = HostContext::default();
-        let scvec0: ScVec = ScVec(vec![ScVal::ScvU32(1)]);
-        let scvec1: ScVec = ScVec(vec![ScVal::ScvU32(1)]);
+        let scvec0: ScVec = ScVec(vec![ScVal::ScvU32(1)].try_into()?);
+        let scvec1: ScVec = ScVec(vec![ScVal::ScvU32(1)].try_into()?);
```

Comment:
It's a-shame we can't avoid the `.try_into()?` because the vec on the left is of const size. 🤔 @graydon is there anyway to take advantage of that?